packages/jest-cli/src/reporters/JestReporterNotify.js

```diff
@@ -70,9 +71,9 @@ class Notifier {
     this.trigger(notification);
   }
 
-  static trigger(notification: Notification): void {
+  trigger(notification: Notification): void {
```

Comment:
I kinda feel like we can go by without this method and just inline it on line #71?